Services/CO.CDP.EntityVerification/appsettings.json

```diff
@@ -5,5 +5,13 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "QueueUrl": "https://localhost.localstack.cloud:4566/000000000000/create-organisation",
+  "ServiceURL": "http://localhost:4566",
+  "AuthenticationRegion": "eu-west-1",
+  "MaxNumberOfMessages": 10,
+  "WaitTimeSeconds": 20,
+  "ConnectionStrings": {
+    "EvDatabase": "Server=localhost;Database=cdp;Username=cdp_user;Password=cdp123;"
+  }
```

Comment:
Can we keep these settings empty in `appsettings.json` and populate them in `appsettings.Development.json`?

Reply:
Done.

Reply:
Done